demo/src/app/components/pagination/demos/customization/pagination-customization.html

```diff
@@ -6,4 +6,35 @@
 </ngb-pagination>
 <hr>
 
+<p>A pagination with customized pages:</p>
+<ngb-pagination [collectionSize]="70" [(page)]="page" [boundaryLinks]="true">
+    <ng-template ngbPaginationPages let-page="page" let-pages="pages">
+        <li class="ngb-custom-pages-item" *ngIf="pages.length > 0">
+            <div class="form-group d-flex flex-nowrap">
+            <label
+                id="paginationInputLabel"
+                for="paginationInput"
+                class="col-form-label mr-2 ml-1"
+            >Pages</label>
+            <input
+                type="text"
+                inputmode="numeric"
+                pattern="[0-9]*"
+                #myInputPage
+                class="form-control custom-pages-input"
+                id="paginationInput"
+                [value]="page"
+                (keyup.enter)="applyPage(myInputPage)"
```

Comment:
can't we pass `(page: string)` and not `(myInputPage: HTMLInputElement)` here for simplicity?

so it would be something like `#inputPage (keyUp.enter) ="applyPage(inputPage.value)"`